src/commands/show.ts

```diff
@@ -79,4 +81,83 @@ export const show = async (): Promise<void> => {
   console.log("  Total Staked Value (SOL) ", tvlStaked.toLocaleString())
   console.log("  Total Liquidity-Pool (SOL) ", tvlLiquidity.toLocaleString())
   console.log("  TVL (SOL) ", (tvlStaked + tvlLiquidity).toLocaleString())
+
+  if ('list' in options) {
+    console.log()
+    console.log("  Validator_manager_authority", state.validatorSystem.managerAuthority.toBase58())
+    console.log(`  Stake list account: ${state.stakeSystem.stakeList.account} with ${state.stakeSystem.stakeList.count}/${"?"} stakes`)
+    console.log("-----------------")
+    console.log("-- Validators ---")
+    console.log(`  Total staked: ${MarinadeUtils.lamportsToSol(state.validatorSystem.totalActiveBalance)} SOL`)
+    console.log(`  List account: ${state.validatorSystem.validatorList.account} with ${state.validatorSystem.validatorList.count}/${"?"} validators`)
+    console.log("-------------------------------------------------------------")
+
+    const epochInfo = await marinadeState.epochInfo()
+    const validatorAccounts = await marinadeState.getValidatorRecords()
+    const stakeInfos = await marinadeState.getStakeInfos()
+
+    let totalStaked = new BN(0)
+    let totalStakedFullyActivated = new BN(0)
+
+    validatorAccounts
+        .forEach((validator, validatorIndex) => {
+      if (validator.activeBalance.toNumber() <= 0) return
+
+      const validatorStakes:StakeInfo[] = stakeInfos
```

Comment:
can you please add a comment indicating what ar you filtering?
also put this filter first, that is the more restrictive: `.filter(stakeInfo => stakeInfo.stake.Stake?.stake.delegation.voterPubkey.toBase58() == validator.validatorAccount.toBase58())`